oligo_designer_toolsuite/oligo_selection/_generate_oligosets.py

```diff
@@ -71,7 +72,8 @@ def apply(
         self,
         oligo_database: OligoDatabase,
         sequence_type: _TYPES_SEQ,
-        n_sets: int = 50,
+        pre_filter: bool,
```

Comment:
I see it more like an additional step. The policy is more like for how we select, and the pre-filtering is for the initial set basically. It would also make sense to put it in the init though. No strong preferences for me. Would you prefer in the Policy instead?

Reply:
I would prefer in the heuristic